src/App.js

```diff
@@ -1,39 +1,40 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+import React, {useState} from "react";
 import "./App.css";
 import BottomRow from "./BottomRow";
 
 function App() {
   //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
-
+  const [hscore, setHScore] = useState(0);
+  const [ascore, setAScore] = useState(0);
```

Comment:
Nice job on using descript variables developing a habit to do this now will pay off later as it allows you to understand your own code while reviewing updating and troubleshooting ect to understand it